src/directives.ts

```diff
@@ -39,7 +39,8 @@ function setupInfiniteScroll(el: DestroyHTMLElement, binding: DirectiveBinding)
   const action = params.action;
 
   const handler = (): void => {
-    if (el.scrollTop >= el.clientHeight - tolerance) {
+    const height = (el.firstChild as DestroyHTMLElement).clientHeight;
```

Comment:
Couple of small things:
1. `firstChild` may be null here. 
2. should be sufficient to cast to HTMLElement.


Though, could you explain why it's necessary to include the height of the first child in the calculation?